### PR TITLE
Added drag&bot company

### PR DESCRIPTION
**Pull request by witalij-s**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,8 @@ A public list of companies that are known to use the Robot Operating System (ROS
...
 | [DOGU](http://www.dogu.xyz/) | DOGU has developed own autonomous robot solution and security specialized AI. We utilize fully automated robot and A.I. for reinventing security service. Outdoor Robot Application Case: Seoul Children's Grand Park and Indoor Robot. DOGU provides ROS 1 and ROS 2 support [^233]. | 2017 | 
 | [Dorabot](https://www.dorabot.com/) | Utilize cutting-edge robotics and AI, such as computer vision, motion planning, mobility, and deep learning, to create automated warehouse solutions. ROS drivers (see [`dorabot`](https://github.com/dorabot)). | 2015 |
-| [Doosan](https://www.doosan.com/en) | Build power plants and industrial facilities, machine tools and automation systems, engines, bridges and construction equipment.  ROS Drivers for communicating with Doosan robots (see [`doosan-robotics`](https://github.com/doosan-robotics)). | 2012 | 
+| [Doosan](https://www.doosan.com/en) | Build power plants and industrial facilities, machine tools and automation systems, engines, bridges and construction equipment.  ROS Drivers for communicating with Doosan robots (see [`doosan-robotics`](https://github.com/doosan-robotics)). | 2012 |
```
Comment (vmayoral) [inline README.md:110]:
```suggestion
| [Doosan](https://www.doosan.com/en) | Build power plants and industrial facilities, machine tools and automation systems, engines, bridges and construction equipment.  ROS Drivers for communicating with Doosan robots (see [`doosan-robotics`](https://github.com/doosan-robotics)). | 2012 | 
```

**Comment by vmayoral** [comment]
Thanks @witalij-s, cleaned up your PR and left only the drag&bot contribution. Definitely makes sense to include it. Only, can you elaborate a bit more on the ROS usage? (see other PRs, for example https://github.com/vmayoral/ros-robotics-companies/pull/28, if you need an intuition on what to add)

**Comment by witalij-s** [comment]
Of course. See the latest commit. Added some more description on ROS usage.

**Comment by vmayoral** [comment]
Welcome in!

**Comment by pablo-quilez** [comment]
Thanks Víctor!

**Pull request merged**